x/epochstorage/keeper/params.go

```diff
@@ -139,6 +139,16 @@ func (k Keeper) GetNextEpoch(ctx sdk.Context, block uint64) (nextEpoch uint64, e
 	return nextEpoch, erro
 }
 
+func (k Keeper) GetCurrentNextEpoch(ctx sdk.Context) (nextEpoch uint64) {
```

Comment:
i this is not valid if epoch size changes? can we have a test for that

Reply:
fixed in next pr